nautobot/dcim/views.py

```diff
@@ -2384,7 +2389,10 @@ def get(self, request, *args, **kwargs):
         if "termination_b_rack" not in initial_data:
             initial_data["termination_b_rack"] = getattr(obj.termination_a.parent, "rack", None)
 
-        form = self.model_form(instance=obj, initial=initial_data)
+        if kwargs.get("termination_b_type", "") == "interface":
+            form = self.model_form(exclude_id=kwargs.get("termination_a_id", ""), instance=obj, initial=initial_data)
```

Comment:
Shouldn't need the default `""` on either of these `get()` calls, and on the second call defaulting to `None` would be more correct in any case:

```suggestion
        if kwargs.get("termination_b_type") == "interface":
            form = self.model_form(exclude_id=kwargs.get("termination_a_id"), instance=obj, initial=initial_data)
```

Reply:
Yeah I didn't really think that bit through properly. Thanks!